tests/software/bowtie/build/test.yml

```diff
@@ -1,4 +1,4 @@
-- name: bowtie build
+- name: bowtie build test
```

Comment:
I thought we want to keep this minimal (without obvious 'test')

Reply:
Good spot 👍🏽